src/window.rs

```diff
@@ -293,24 +294,11 @@ impl WindowWrapper {
     }
 
     pub fn draw_frame(&mut self) -> bool {
-        if let Ok(new_size) = LogicalSize::new(&self.window) {
-            if self.previous_size != new_size {
-                handle_new_grid_size(new_size, &self.renderer);
-                self.previous_size = new_size;
-            }
-        }
-
-        if let Ok(new_dpis) = dpis(&self.window) {
-            if self.previous_dpis != new_dpis {
-                let physical_size = PhysicalSize::new(&self.window);
-                self.window
-                    .set_size(
-                        (physical_size.width as f32 * new_dpis.0 / self.previous_dpis.0) as u32,
-                        (physical_size.height as f32 * new_dpis.1 / self.previous_dpis.1) as u32,
-                    )
-                    .unwrap();
-                self.previous_dpis = new_dpis;
-            }
+        let sdl_window = Sdl2Window::new(&self.window);
```

Comment:
You use the name sdl_window to refer to 2 concepts. Above, its the actual sdl window and here its the sdl window wrapper. I'm curious if a better name could be used here to make sure its obvious that this is the wrapper?

Reply:
wrapper_window?

Reply:
Looking back, 

I would use window as before for the ...subsystem and the sdl_window for the Sdl2Window::new.

This is a very good point, needs changing before accepted.